test/Microsoft.Sbom.Api.Tests/Executors/LicenseInformationFetcherTests.cs

```diff
@@ -16,19 +17,20 @@ namespace Microsoft.Sbom.Api.Executors.Tests;
 public class LicenseInformationFetcherTests
 {
     private readonly Mock<ILogger> mockLogger = new Mock<ILogger>();
+    private readonly Mock<IRecorder> mockRecorder = new Mock<IRecorder>();
     private readonly Mock<HttpClient> mockHttpClient = new Mock<HttpClient>();
     private Mock<LicenseInformationService> mockLicenseInformationService;
 
     [TestInitialize]
     public void Setup()
     {
-        mockLicenseInformationService = new Mock<LicenseInformationService>(mockLogger.Object, mockHttpClient.Object);
```

Comment:
Rather than create mocks just to pass to a mock I tend to make an `ILicenseInformationService` which is what gets passed around. That also frees you to make `LicenseInformationService` into `ClearlyDefinedLicenseInformationService` and given the proper abstractions have other stores be configurable drop-ins. It can sometimes get a bit "Java-y" but I find that if you think about what methods go on the interface (and what parameters they have) it can help you realize what the right abstractions are.